src/GoogleAnalyticsTracker.js

```diff
@@ -179,6 +179,13 @@ export class GoogleAnalyticsTracker {
     GoogleAnalyticsBridge.setClient(this.id, clientId);
   }
 
+  /**
+   * Get the current clientId.
+   */
+  getCID(callback) {
+    GoogleAnalyticsBridge.getClientId(this.id, callback);
```

Comment:
JS should return Promise, [JS example](https://github.com/idehub/react-native-google-analytics-bridge/blob/d90d15316f0df843fd2f3063569aec18824990de/src/GoogleAnalyticsTracker.js#L260)